src/api/transaction/sign.js

```diff
@@ -48,12 +48,14 @@ function computeSignature(txJSON, keypair) {
 }
 
 function sign(txJSON: {Account: string; SigningPubKey: string,
-    TxnSignature: string}, secret: string):
+    TxnSignature: string}, secret: string, isRegular = false):
```

Comment:
Seed.from_json will parse passphrases too. In case you strictly want to
accept bs58 only ....